orbital-mechanics/Lecture4/Lecture4.ipynb

```diff
@@ -0,0 +1,397 @@
+{
```

Comment:
<p>Please rewrite this based on the following comments instructions. This is because with mitansh's help, we found out that the website doesn't pick up the hyperlinks. So these need to be written differently so that html conversion works correctly: <a href="https://github.com/angadhn/SpacecraftDynamics/pull/3#discussion_r1679994551" target="_blank">https://github.com/angadhn/SpacecraftDynamics/pull/3#discussion_r1679994551</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>